physics/module_MYNNPBL_wrapper.F90

```diff
@@ -204,7 +214,7 @@ SUBROUTINE mynnedmf_wrapper_run(        &
 
 !MYNN-1D
       REAL(kind=kind_phys), intent(in) :: delt, dtf
-      INTEGER, intent(in) :: im, levs
+      INTEGER, intent(in) :: im, ix, levs
```

Comment:
ix isn't used in the code, please don't add it. The recent cleanup when making the ccpp-framework changes disallow passing horizontal_dimension to _run routines.

Reply:
taken out